frontend-react/src/pages/submissions/SubmissionsTable.tsx

```diff
@@ -49,6 +110,39 @@ function SubmissionsTable() {
                         })}
                     </tbody>
                 </table>
+                {submissions.length === 0 && (
+                    <p>There were no results found.</p>
+                )}
+                {(submissions.length > 0 || paginationCursor) && (
```

Comment:
why ` || ` logic here? Is it because length maybe zero for the last page and `paginationCursor` may be null on the first? This does mean that submissions.length could be zero above when you access [-1]?